Makefile

```diff
@@ -1,3 +1,35 @@
+# configurations
+
+OUTPUT_DIR = ./output
+INFO_TXT_FILE = info.txt
+
+SDK_MAIN_FILE = ./code/AdobeEdge.brs
+SDK_COMPONENTS_DIR = ./code/components
+
+GIT_HASH:=$(shell git show --name-status | grep commit | awk '{print $$2}' | head -c6)
+VERSION:=$(shell grep "VERSION = " ./code/AdobeEdge.brs | sed -e 's/.*= //; s/,//g; s/"//g')
+
 # bsc: https://github.com/rokucommunity/brighterscript
 install-bsc:
-	(npm install brighterscript -g)
\ No newline at end of file
+	(npm install brighterscript -g)
+
+clean:
+	(rm -rf $(OUTPUT_DIR))
+	
+packing:clean
+	@echo "######################################################################"
+	@echo "##### packing"
+	@echo "######################################################################"
+	@echo git-hash=$(GIT_HASH)
+
+	mkdir -p $(OUTPUT_DIR)
```

Comment:
As discussed we could generate the zip with the version number. And we include sample app with the final build of the SDK.

Sample app changes script can be added after the alpha release.

Reply:
@addb  https://github.com/adobe/aepsdk-roku/issues/25 is created to track the task.